packages/renderer/src/lib/kubernetes-port-forward/PortForwardActions.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+import { faSquareUpRight, faTrash } from '@fortawesome/free-solid-svg-icons';
+
+import { withConfirmation } from '/@/lib/dialogs/messagebox-utils';
+import type { PortForwardRow } from '/@/lib/kubernetes-port-forward/port-forward-row';
+import { kubernetesCurrentContextPortForwards } from '/@/stores/kubernetes-contexts-state';
+import { type UserForwardConfig, WorkloadKind } from '/@api/kubernetes-port-forward-model';
+
+import ListItemButtonIcon from '../ui/ListItemButtonIcon.svelte';
+
+interface Props {
+  object: PortForwardRow;
+}
+let { object }: Props = $props();
+
+let userConfigForward: UserForwardConfig | undefined = $derived(
+  $kubernetesCurrentContextPortForwards.find(
+    forward =>
+      forward.kind === WorkloadKind.POD && forward.name === object.name && forward.namespace === object.namespace,
+  ),
+);
+
+async function deletePortForward(): Promise<void> {
+  if (!userConfigForward) return;
+
+  await window.deleteKubernetesPortForward(userConfigForward, object.mapping);
+}
+
+async function openExternal(): Promise<void> {
+  return window.openExternal(`http://localhost:${object.mapping.localPort}`);
```

Comment:
`http://localhost:`` I feel should be a const at the top of the file

Reply:
why ?

Reply:
Just a suggestion in case for any other scenarios